docs/examples/BasicSingle.tsx

```diff
@@ -10,86 +10,65 @@ const Checkbox = ({ children, ...props }: JSX.IntrinsicElements['input']) => (
   </label>
 );
 
-interface State {
-  readonly isClearable: boolean;
-  readonly isDisabled: boolean;
-  readonly isLoading: boolean;
-  readonly isRtl: boolean;
-  readonly isSearchable: boolean;
-}
+export default () => {
+  const [isClearable, setIsClearable] = useState(false);
```

Comment:
`isClearable` and `isSearchable` should start off as `true`

Reply:
good catch 👍